analyses/count-contributions/02-format-contributions.Rmd

```diff
@@ -151,7 +151,11 @@ count_contributions_df <- count_contributions_df %>%
   mutate(author = recode_factor(author, !!!name_recoding)) %>%
   # If people had two names/IDs, this will de-duplicate now that things have 
   # been recoded
-  distinct()
+  distinct() %>%
+# Add Yang Yang to the list for contributions to the `chromothripsis` module. This was renamed from
+# `sv-analysis` since those contributions, so those were missed. [Ticket](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1599) and [rename PR](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1069)
+  add_row(module = "chromothripsis", author = "Yang Yang") %>%
```

Comment:
Flipping order w/ distinct in case any new contributions come in.
```suggestion
  # Add Yang Yang to the list for contributions to the `chromothripsis` module. This was renamed from
  # `sv-analysis` since those contributions, so those were missed. [Ticket](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1599) and [rename PR](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1069)
  add_row(module = "chromothripsis", author = "Yang Yang") %>%
  distinct() %>%
```

Reply:
ok, sounds good - I will revert to the old html/results and update order